wee_slack.py

```diff
@@ -704,15 +704,24 @@ def buffer_input_callback(signal, buffer_ptr, data):
     if not channel:
         return w.WEECHAT_RC_ERROR
 
-    reaction = re.match("^(\d*)(\+|-):(.*):\s*$", data)
-    substitute = re.match("^(\d*)s/", data)
+    def get_id(message_id):
+        if not message_id or len(message_id) == 0:
+            return 1
+        elif message_id[0:2] == "0x":
+            return message_id
+        else:
+            return int(message_id)
+
+    message_id = "(\d*|0x[0-9a-fA-F]{12})"
```

Comment:
My IDs are not 12 chars long. they are the format `0x6c3d3900`

Reply:
mine too.

Reply:
I’m waiting for the dependent PR to be accepted before doing any change here, feel free to modify it locally in the meantime :)

Reply:
I’m waiting for the dependent PR to be accepted before doing any change here (but the maintainer seems busy with other things at the moment), feel free to modify it locally in the meantime :)